drivers/include/net/netdev/ieee802154.h

```diff
@@ -136,7 +136,6 @@ typedef struct {
  * @brief   Received packet status information for IEEE 802.15.4 radios
  */
 typedef struct netdev_ieee802154_rx_info {
-    uint64_t timestamp;     /**< Timestamp value of a received frame in ns */
```

Comment:
the `at86rf215` driver uses this member regardless of MODULE_SOCK_AUX_TIMESTAP.

On the other side (could also be follow ups):
1. MODULE_SOCK_AUX_TIMESTAP -> MODULE_SOCK_AUX_TIMESTAMP
2. Should we rather model this as a `netdev` feature rather than `sock`? It's not uncommon to have radios that timestamp frames and it's definitely useful for the MAC. I think it would make sense to add a `MODULE_NETDEV_IEEE802154_RX_TIMESTAMP` that gets selected by `MODULE_SOCK_AUX_TIMESTAMP`.

Reply:
Like this?

Reply:
yes! exactly